EDSAnalytics/EDSTest/azure-pipeline.yml

```diff
@@ -0,0 +1,72 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/EDSAnalytics'
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/EDSAnalytics'
+
+schedules:
+  - cron: '0 0 * * 0'
+    displayName: Weekly build
+    branches:
+      include:
+        - master
+    always: true
+
+variables:
+  - template: '/miscellaneous/build_templates/variables.yml'
+  - name: projPath
+    value: '/EDSAnalytics/EDSTest'
+  - name: testPath
+    value: '/EDSAnalytics/EDSTestTest'
+  - name: CredFile
+    value: appsettings.json
+  - name: CredFileLibrary
+    value: product-readiness.sds_dotnet_rest.json
+  - name: covStream
+    value: SDSDotNetAPI
+  - name: bdProject
+    value: SDS_DotNet_REST
+
+jobs:
+  - job: Tests
+    strategy:
+      matrix:
+        linux:
+          imageName: 'ubuntu-16.04'
+        windows:
+          imageName: 'vs2017-win2016'
+    pool:
+      vmImage: $(imageName)
+    steps:
+      - template: '/miscellaneous/build_templates/config.yml'
+        parameters:
+          fileOriginal: $(CredFile)
+          fileLibrary: $(CredFileLibrary)
+          filePath: $(projPath)
+
+      - script: |
+          dotnet restore
+          dotnet msbuild -p:HIGHENTROPYVA=true
+        workingDirectory: $(Build.SourcesDirectory)$(projPath)/
+        displayName: 'Build app'
+
+      - script: |
+          dotnet restore
+          dotnet test
+        workingDirectory: $(Build.SourcesDirectory)$(testPath)/
+        displayName: 'Run tests'
+
+  - template: '/miscellaneous/build_templates/analysis.yml'
```

Comment:
This references a file that has not yet been added to the repository. You can copy it (and the other files in the `/miscellaneous/build_templates/` folder) from [here](https://github.com/osisoft/OSI-Samples-OMF/blob/master/miscellaneous/build_templates/analysis.yml). Without all three files (config/variables/analysis) this build pipeline won't work.

Reply:
Done.